apps/public-docsite/src/components/ReactIconGrid/ReactIconGrid.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import * as stylesImport from './ReactIconGrid.module.scss';
+import { IIconGridProps } from './IconGridProps.types';
+import { FluentIconsProps } from '@fluentui/react-icons';
+const styles: any = stylesImport;
+
+export const ReactIconGrid = (iconGridProps: IIconGridProps) => {
+  const [search, setSearch] = React.useState('');
+  const [size, setSize] = React.useState('');
+
+  const _onSearchQueryChanged = (ev: React.ChangeEvent<HTMLInputElement>) => {
+    setSearch(ev.currentTarget.value);
+  };
+
+  const _getItems = (props: IIconGridProps): React.FC<FluentIconsProps>[] => {
+    const { icons } = props;
+
+    return icons.filter(
+      icon =>
+        icon &&
+        icon.displayName &&
+        icon.displayName.toLowerCase().indexOf(search.toLowerCase()) !== -1 &&
+        icon.displayName.indexOf(size) !== -1,
+    );
+  };
+
+  const _filterBySize = (ev: React.ChangeEvent<HTMLInputElement>) => {
+    const newSize = ev.currentTarget.value;
+    newSize === 'All' ? setSize('') : setSize(newSize);
+  };
+
+  const _renderIcon = (Icon: React.FC<FluentIconsProps>): JSX.Element => {
+    let iconClassName = `ms-Icon ms-Icon--${Icon.displayName}`;
+
+    return (
+      <li key={Icon.displayName} aria-label={Icon.displayName}>
+        <Icon className={iconClassName} />
+        <span>{Icon.displayName}</span>
+      </li>
+    );
+  };
+
+  const filteredIcons = _getItems(iconGridProps);
+
+  return (
+    <div>
+      <input placeholder="Search icons" value={search} onChange={_onSearchQueryChanged} className={styles.searchBox} />
```

Comment:
Can you use the SearchBox and ChoiceGroup components here? They'll have proper styling and behavior to match the rest of the site.